### PR TITLE
new derived table Hold stat codes

### PR DESCRIPTION
**Pull request by natalyapik**
new derived table, added to run list as well

### CODE REVIEW

**Comment by natalyapik** [comment]
#552

File: sql_metadb/derived_tables/holdings_statistical_codes.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    
+CREATE INDEX ON holdings_statistical_codes (instance_id);
+CREATE INDEX ON holdings_statistical_codes (instance_hrid);
+CREATE INDEX ON holdings_statistical_codes (holdings_id);
+CREATE INDEX ON holdings_statistical_codes (holdings_hrid);
+CREATE INDEX ON holdings_statistical_codes (statistical_code_id);
+CREATE INDEX ON holdings_statistical_codes (statistical_code_type_id);
+CREATE INDEX ON holdings_statistical_codes (statistical_code_type_name);
+CREATE INDEX ON holdings_statistical_codes (statistical_code);
+CREATE INDEX ON hldings_statistical_codes (statistical_code_name);
```
Comment (jenmawe) [inline sql_metadb/derived_tables/holdings_statistical_codes.sql:40]: This is missing an "o", hldings_ instead of holdings_

**Review by jenmawe — APPROVED** [review]

File: sql_metadb/derived_tables/holdings_statistical_codes.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS holdings_statistical_codes;
+CREATE TABLE holding_statistical_codes AS
```
Comment (adoerrer) [inline sql_metadb/derived_tables/holdings_statistical_codes.sql:2]: There is a 's' missing after holding

File: sql_metadb/derived_tables/holdings_statistical_codes.sql [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    sct.name AS statistical_code_name,
+    stc.statistical_code_ordinality 
+FROM 
+    stcodes AS stc
+    LEFT JOIN folio_inventory.statistical_code__t AS sct ON stc.statistical_code_id::uuid = sct.id::uuid 
+    LEFT JOIN folio_inventory.statistical_code_type__t AS sctt ON sct.statistical_code_type_id::uuid = sctt.id::uuid;
+    
+CREATE INDEX ON holdings_statistical_codes (instance_id);
+CREATE INDEX ON holdings_statistical_codes (instance_hrid);
+CREATE INDEX ON holdings_statistical_codes (holdings_id);
```
Comment (adoerrer) [inline sql_metadb/derived_tables/holdings_statistical_codes.sql:34]: It should be `holding_id` like in the main query.

File: sql_metadb/derived_tables/holdings_statistical_codes.sql [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+CREATE INDEX ON holdings_statistical_codes (holdings_hrid);
```
Comment (adoerrer) [inline sql_metadb/derived_tables/holdings_statistical_codes.sql:35]: It should be `holding_hrid` like in the main query.

File: sql_metadb/derived_tables/runlist.txt [hunk 5]
```diff
@@ -6,6 +6,7 @@ agreements_subscription_agreement_org_ext.sql
 finance_transaction_invoices.sql
 finance_transaction_purchase_order.sql
 holdings_ext.sql
+holdings_statistical_codes
```
Comment (adoerrer) [inline sql_metadb/derived_tables/runlist.txt:9]: The suffix `.sql` is missing

**Review by adoerrer — CHANGES_REQUESTED** [review]
I've found some typos to fix. Overall the query looks good and runs great when the typos are fixed, thank you!

**Review by adoerrer — APPROVED** [review]
Query and results look great, thank you!

File: sql_metadb/derived_tables/runlist.txt (hunk 5 above)
Comment (nassibnassar) [reply sql_metadb/derived_tables/runlist.txt:9]: @adoerrer You may have accidentally approved this pull request before the error above was fixed.

Comment (adoerrer) [reply sql_metadb/derived_tables/runlist.txt:9]: Yes, I've missed that. Thank you, for catching that!

**Comment by jenmawe** [comment]
@natalyapik This is really minor. In all our other MM derived tables, holdings is always plural. Did you want to add the "s" here where needed to be consistent? For example, instead of holding_id, have holdings_id.

**Review by adoerrer — APPROVED** [review]

File: sql_metadb/derived_tables/runlist.txt (hunk 5 above)
Comment (nassibnassar) [reply sql_metadb/derived_tables/runlist.txt:9]: This is still not fixed.  I will add a commit to fix it.

**Pull request merged**